core/src/main/java/io/bisq/core/dao/blockchain/parse/OpReturnVerification.java

```diff
@@ -56,6 +56,9 @@ boolean processDaoOpReturnData(Tx tx, int index, long bsqFee,
                         return compensationRequestVerification.processOpReturnData(tx, opReturnData, txOutput,
                                 bsqFee, blockHeight, btcOutput);
                     case DaoConstants.OP_RETURN_TYPE_VOTE:
+                        if (bsqOutput == null) {
```

Comment:
Maybe we add a TODO here as I think it was not allowed to have no bsqOutput here. But voting is WIP anyway...

Reply:
TODO added.